### ISSUE TITLE
Issue with getter for getHomeworld()

### ISSUE THREAD

**Issue by bdschultzAU** [salty-pig/swapi-node#31]
Hello.  I've been playing around your api w/ express.  When I grab a person, then attempt to retrieve their homeworld as shown in the documentation, all I receive is a link.  That's not the expected behavior from what I read in your documentation.   Is this a user error or a bug?  thank you

```
"use strict";

const app = require("express")();
const swapi = require("swapi-node");

const PORT = 5000;

app.get("/people/:id", (req, res) => {
  swapi.getPerson(req.params.id).then(person => {
    person.getHomeworld().then(result => {
      res.json(result);
    });
  });
});

app.listen(PORT, () => console.log("SWAPI sdk app running on port: " + PORT));

```

response:  "https://swapi.co/api/planets/1/"

**Comment by lholmquist** [comment]
@bdschultzAU good question.  I haven't looked at this in a while,  so i'll check on it

**Comment by lholmquist** [comment]
@bdschultzAU sorry for the loooong time this took, but i figured it out,  it was a simple thing.  I'll get a new version out soon™

**Comment by lholmquist** [comment]
@bdschultzAU should be fixed with 0f25ff0d457b5861f6215e5586ebc3ea229f5af8

released as 0.4.4
